### ISSUE TITLE
Invert colors

### ISSUE THREAD

**Issue by ramiel** [PeculiarVentures/GammaCV#51]
Is there a way to invert colors?

**Comment by WorldThirteen** [comment]
@ramiel, thanks for the interest in GammaCV. Yes, there is a way to invert the colors of the image. I have prepared the example: [CodePen](https://codepen.io/WorldThirteen/full/JQqaYo).

I've used the next logic:
1. Create a Tensor that represents the white image
2. Use a math operation `sub`, and sub the input image from the white image, got the inverted colors.

```
 // creates a tensor describes a full white picture;
 const whiteTensor = new gm.Tensor('uint8', [height, width, 4]);
 whiteTensor.data.fill(255);
  
 const operation = gm.sub(whiteTensor, imageTensor);
```
